From 89311d21b0547d4a146062fee35fd2be308936fe Mon Sep 17 00:00:00 2001
From: Nathan <ndowens@artixlinux.org>
Date: Sun, 12 Sep 2021 17:13:14 -0500
Subject: [PATCH] Fix plugins dir

Floods console with lib/ModemManager not found
Add correct location to fix
---
 data/org.freedesktop.ModemManager1.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.freedesktop.ModemManager1.service.in b/data/org.freedesktop.ModemManager1.service.in
index 175f3dd2..08501789 100644
--- a/data/org.freedesktop.ModemManager1.service.in
+++ b/data/org.freedesktop.ModemManager1.service.in
@@ -6,6 +6,6 @@
 
 [D-BUS Service]
 Name=org.freedesktop.ModemManager1
-Exec=@sbindir@/ModemManager
+Exec=@sbindir@/ModemManager --test-plugin-dir=/lib/ModemManager
 User=root
 SystemdService=dbus-org.freedesktop.ModemManager1.service
-- 
2.33.0

